src/pymorize/rule.py

```diff
@@ -284,3 +286,64 @@ def global_attributes_set_on_rule(self):
         result = {attr: getattr(self, attr, None) for attr in attrs}
         result["creation_date"] = creation_date
         return result
+
+    def get_global_attributes(self, cmor_version):
```

Comment:
I would add a class level variable (a tuple) that says which strings for `cmor_version` are allowed, just like we have it on the `CMORizer`.

Reply:
```suggestion
    _SUPPORTED_CMOR_VERSIONS = ("CMIP6", "CMIP7")

    def get_global_attributes(self, cmor_version):
        if cmor_version not in _SUPPORTED_CMOR_VERSIONS:
            raise ValueError(f"{cmor_version} not supported! Currently only f{_SUPPORTED_CMOR_VERSIONS=} can be used!")
```